src/applications/disability-benefits/all-claims/content/unemployabilityDates.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
+
+export const dateDescription = (
+  <div>
+    <h5>Disability Dates</h5>
```

Comment:
Is there a difference between defining this and just using `'ui:description': 'Disability Dates'` directly in the uiSchema?